lib/WebLedgerClient.js

```diff
@@ -238,34 +248,36 @@ class WebLedgerClient {
    * @returns {Promise<Object>} The result of the send.
    */
   async sendDocument({document, service, headers = {}}) {
-    const requestHeaders = Object.assign(
-      {}, constants.DEFAULT_HEADERS, headers);
     if(!this.ledgerAgent) {
       await this.getAgent();
     }
     const baseURL = this.ledgerAgent.service[service];
-    const serviceApi =
-      create({baseURL, httpsAgent: this.httpsAgent, timeout: this.timeout});
-    const response = await serviceApi.post('/', document, {
-      headers: requestHeaders
-    });
-    if(response.problem) {
+    try {
+      const response = await httpClient.post('', {
```

Comment:
Is the URL supposed to be an empty string?

Reply:
Ah, I do see the `prefixUrl: baseURL,` below, I'm assuming the empty string is just not adding anything else on to the baseURL.

Reply:
I could try it with out a string at all, but yes in this case there is a prefixUrl defined, that means ky will add `/` at the end of the URL. That replicates the behavior axios was doing previously and ensures the nock tests continue to function.

Reply:
As I mentioned earlier, to avoid confusion, as evidenced by Mandy's comment, please don't use the `prefixUrl` API.

Reply:
prefixUrl has been removed here: https://github.com/digitalbazaar/web-ledger-client/pull/25/commits/3d05d1ee6e6fe73289555a5cffdc45c75b204857